### PR TITLE
Fix for check arguments not being converted (from string) to int/float/etc (C4-1030)

### PR DESCRIPTION
**Pull request by dmichaels-harvard**

### CODE REVIEW

File: react/src/pages/AccountsComponent.js [hunk 1]
```diff
@@ -2,14 +2,34 @@ import React, { useEffect, useState } from 'react';
 import { BarSpinner, StandardSpinner } from '../Spinners';
 import { useSearchParams } from 'react-router-dom';
 import Char from '../utils/Char';
+import Client from '../utils/Client';
```
Comment (dmichaels-harvard) [inline react/src/pages/AccountsComponent.js:5]: These changes are unrelated to this fix - just a minor UI update to the accounts page to include a link to the SSL certificate info for Portal and Foursight, and to include the portal access key ID for the current account.

**Review by willronchetti — APPROVED** [review]
Looks good just needs real version
